scripts/annotate-coments.js

```diff
@@ -12,32 +12,64 @@ function mapComment(comment, annotations) {
     assetHeadline: comment.assetHeadline,
     voteRating: comment.voteRating,
     voteCount: comment.voteCount,
-    // replies: mapComment() : TODO: replies
     entities: annotations.entities,
     sentiment: annotations.sentiment,
     sentences: annotations.sentences
-  }
+  };
 }
 
-module.exports = (comments) => {
-  const annotatedData = [];
-  let count = 0;
-
-  comments.forEach((comment) => {
-    languageClient.annotate(comment, (err, annotations) => {
+function annotateComment(comment) {
+  return new Promise((res, rej) => {
+    languageClient.annotate(comment, (err, results) => {
       if (err) {
-        console.log(err);
+        rej(err);
       } else {
-        annotatedData.push(mapComment(comment, annotations));
-      }
-
-      count++;
-      if (count === comments.length) {
-        fs.writeFileSync(`test/fixtures/annotated-comments-${comment.assetId}.json`, JSON.stringify(annotatedData));
+        res(mapComment(comment, results));
       }
     });
   });
+}
+
+function annotateComments(comments) {
+  return Promise.all(comments.map(annotateComment));
+}
+
+function annotateCommentAndReplies(comment) {
```

Comment:
I didn't realise replies can have replies.. this only handles one level - need to do it recursively.

Though, I haven't inspected whether replies are also included in the response as normal comments (I would assume they aren't, but I could see it happening)